index.html

```diff
@@ -2026,6 +2026,34 @@
       </p>
 
       <section>
+        <h3>Cryptography Suites and Libraries</h3>
+
+        <p>
+The entire data model described in this specification is protected through the
+use of cryptography. Implementers should be aware of the underlying
+cryptography suites and libraries that are used to implement the
+creation and verification of digital signatures and mathematical proofs
+utilized by their systems when processing <a>credentials</a> and
+<a>profiles</a>. Software developers with extensive experience implementing
+or auditing systems that use cryptography must be used to ensure that
+systems are properly designed. Proper
+<a href="https://en.wikipedia.org/wiki/Red_team">red teaming</a>
+is also suggested to remove bias from security reviews.
+        </p>
+
+        <p>
+Cryptography suites and libraries have a shelf life and eventually fall to
+new attacks and technology advances. Any production quality system must take
+this reality into account and ensure that mechanisms exist to easily upgrade
+old or broken cryptographic suites and libraries in a proactive manner.
```

Comment:
should also mention needing process for revoking or updating credentials themselves as well in this case.

Reply:
Done.